backend/Resources/MetadataExperiencePostMappingResource.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections.Generic;
+using System;
+
+using OpenData.Resources;
+
+namespace OpenData.Resources
+{
+	public class MetadataExperiencePostMappingResource
+    {
+    	//public ExperiencePostTitleResource ExperiencePost { get; set; }
```

Comment:
Do we want these comments on development?

Reply:
no, we do not. Its fixed!